docs/operations/backups-and-restores-with-kastenk10.md

```diff
@@ -194,7 +205,7 @@ a consistent snapshot can be taken. It serves no other purpose. Notice how this
 container must have the `privileged` flag set to `true`. This is necessary to run
 the `fsfreeze` command.
 
-## 5.2: Adding pre/post-snapshot hooks to quiesce the application/filesystem
+#### Step 5.2 - Adding Pre/Post-Snapshot Hooks To Quiesce The Application/Filesystem
```

Comment:
- Add (instead of Adding)
- the (lower case)

Reply:
fixed